trusted_os/main.go

```diff
@@ -168,6 +168,19 @@ func main() {
 		log.Printf("Failed to determine OS MMC block (no OS installed?): %v", err)
 	}
 
+	newIdentity, err := newWitnessIdentity(Storage)
+	if err != nil {
+		log.Printf("Failed to read new witness identity MMC block: %v", err)
+	}
+	if newIdentity {
```

Comment:
Don't forget to clear the "newWitnessIdentity" MMC block so we don't get stuck incrementing the counter on each boot!